panasonic_camera/camera_manager.py

```diff
@@ -47,6 +50,13 @@ def _connect(self):
             logger.debug(
                 'Connect to {}: {}'.format(device.friendly_name, hostname))
             self.camera = PanasonicCamera(hostname)
+            # If we have a _identify_as property, assume this is a camera like
+            # Panasonic DC-FZ80 which requires registering the remote control
+            # device (in this case, us) with the camera first.
+            if self._identify_as:
+                logger.debug(f'Attempting to identify as {self._identify_as}')
+                self.camera.register_with_camera(
+                    identify_as=self._identify_as)
```

Comment:
This fits in a single line with 80 characters (allowed maximum), i.e.

```python
self.camera.register_with_camera(identify_as=self._identify_as)
```

Reply:
fixed in f27ac81dcf4040cf0e4d65251b5322a3fcd4c1ae